config/locales/en.yml

```diff
@@ -21,3 +21,4 @@
 
 en:
   hello: "Hello world"
+  badge: "Crachá"
```

Comment:
@frmendes If this was under `pt` wouldn't the locale have to be changed for it to work properly. If the locale was changed we would encounter the missing translations error again 🤔